romfs/lang/de.json

```diff
@@ -98,46 +98,46 @@
             "complete": "Installation abgeschlossen",
             "desc0": " Dateien erfolgreich installiert!",
             "desc1": " installiert!",
-            "downloading": "Läd ",
+            "downloading": "Lädt ",
             "at": " herunter von "
         },
         "nca_verify": {
-            "title": "Ungültige NCA-Signature erkannt!",
-            "desc": "Unsachgemäß signierte Software sollte nur von vertrauenswürdigen Quellen\ninstalliert werden. Dateien, die Cartridge-Repacks und DLC-Freischaltgeräte enthalten,\nwerden immer diese Warnung anzeigen. Sie können diese Prüfung in den Einstellungen des Awoo Installers deaktivieren.\nnSind Sie sicher, dass Sie die Installation fortsetzen möchten?",
+            "title": "Ungültige NCA-Signatur erkannt!",
+            "desc": "Unsachgemäß signierte Software sollte nur von vertrauenswürdigen Quellen\ninstalliert werden. Dateien von gepackten Spiele Modulen und DLC-Freischaltung enthalten,\nwerden immer diese Warnung anzeigen. Sie können diese Prüfung in den Einstellungen des Awoo Installer's deaktivieren.\nSind Sie sicher, dass Sie die Installation fortsetzen möchten?",
             "opt1": "Ja, ich verstehe die Risiken",
-            "error": "Die gewünschte NCA ist nicht valide signiert: "
+            "error": "Die gewünschte NCA ist nicht richtig signiert: "
         },
         "finished": [
-            "Geniese deine \"legalen Sicherheitskopien\"!",
-            "Ich bin mir sicher nachdem du das Spiel probiert hast wirst du Spaß dabei haben das spiel tatsächlich zu kaufen!",
-            "Dul halt das Spiel sichel gekauft? Nintendo-san dankel dil fül deinen Kauf!",
-            "DRM zu umgehen ist toll, stimmts?",
-            "Du hast wahrscheinlich sechs Bäume gerettet dadurch dass du das Spiel nicht gekauft hast! All das Plastik geht irgendwo hin!",
+            "Genieße deine \"legalen Sicherheitskopien\"!",
+            "Ich bin mir sicher, nachdem du das Spiel probiert hast. Wirst du Spaß dabei haben, das Spiel tatsächlich zu kaufen!",
```

Comment:
eh?

Reply:
what do you want to say?

Reply:
The sentence doesn't make any sense split that way.

Reply:
So keep the original? Only with upper "Spiel"
"Ich bin mir sicher nachdem du das Spiel probiert hast wirst du Spaß dabei haben das Spiel tatsächlich zu kaufen!"